test/dlc_tests/sanity/test_binary_visibility.py

```diff
@@ -16,8 +17,10 @@ def test_binary_visibility(image: str):
     'https://' may still be private, codebuild 'build' job uses 'curl' i.e. unsigned request to fetch them and hence should
     fail if an 'https://' link is still private
     """
-    if is_trcomp_image(image):
-        pytest.skip("Skipping test for HF TrComp images due to labels from base image being carried over")
+
+    if is_trcomp_image(image) and re.search('tensorflow', image):
```

Comment:
Please use the `get_framework_and_version_from_tag` function:
https://github.com/aws/deep-learning-containers/blob/cb8b56efdb94e16bc41fdeb81c90a75260ff1f16/test/test_utils/__init__.py#L1202-L1208

and also ensure that this test is skipped specifically when the framework version is `2.6.*` using the `packaging.version.Version` and `packaging.specifiers.SpecifierSet` functions.